src/bin/zkai_stwo_proof_size_estimate.rs

```
@@ -0,0 +1,208 @@
+use std::fs;
```

Clippy (1.95.0)

[warning] unused import

(warning)